packages/grafana-ui/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -84,18 +85,9 @@ export class ErrorBoundaryAlert extends PureComponent<WithAlertBoxProps> {
                 </details>
               </Alert>
             );
-          } else {
-            return (
-              <div className={getStyles()}>
-                <h2>{title}</h2>
-                <details style={{ whiteSpace: 'pre-wrap' }}>
-                  {error && error.toString()}
-                  <br />
-                  {errorInfo.componentStack}
-                </details>
-              </div>
-            );
           }
+
+          return <ErrorWithStack className={getStyles()} title={title || ''} error={error} errorInfo={errorInfo} />;
```

Comment:
Instead of passing styles by name, can we move them to the ErrorWithStack component?

Reply:
Yes, of course!